app/javascript/tests/enrollment/steps/PublicHealthManagement.test.js

```diff
@@ -59,12 +60,16 @@ describe('PublicHealthManagement', () => {
 
     expect(wrapper.find('#exposure_risk_assessment').exists()).toBe(true);
     expect(wrapper.find('#exposure_risk_assessment').prop('value')).toEqual('');
+    expect(wrapper.find('[htmlFor="exposure_risk_assessment"]').find(InfoTooltip).exists()).toBe(true);
```

Comment:
very small but would rather this find be `wrapper.find({ controlId: 'exposure_risk_assessment' }).find(InfoTooltip)` for test consistency